features/mbedtls/targets/TARGET_STM/TARGET_STM32F4/TARGET_NUCLEO_F439ZI/mbedtls_device.h

```diff
@@ -22,5 +22,7 @@
 
 #define MBEDTLS_AES_ALT
 
+#define MBEDTLS_MD5_ALT
 
+#define MBEDTLS_MD5_C
```

Comment:
@adustm This is application configuration, so it should be defined by user in mbed_app.json ot the User defined configuration files